native/cocos/editor-support/spine/Atlas.cpp

```diff
@@ -51,7 +51,7 @@ Atlas::Atlas(const String &path, TextureLoader *textureLoader, bool createTextur
     const char *lastSlash = lastForwardSlash > lastBackwardSlash ? lastForwardSlash : lastBackwardSlash;
     if (lastSlash == path) lastSlash++; /* Never drop starting slash. */
     dirLength = (int)(lastSlash ? lastSlash - path.buffer() : 0);
-    dir = SpineExtension::calloc<char>(dirLength + 1, __FILE__, __LINE__);
+    dir = SpineExtension::calloc<char>(dirLength + 1, __SPINE_FILE__, __SPINE_LINE__);
```

Comment:
__SPINE_FILE__ is defined to empty string for emscripten build. That will save some bytes.